cupyx/scipy/sparse/linalg/_interface.py

```diff
@@ -0,0 +1,679 @@
+import warnings
+
+import cupy as cp
+
+from cupyx.scipy.sparse import isspmatrix
```

Comment:
Could you avoid importing non-module symbols? ([coding guidelines](https://docs.cupy.dev/en/stable/contribution.html#coding-guidelines))

Reply:
done ✌🏾  😅